gprofiler/main.py

```diff
@@ -815,6 +890,11 @@ def main() -> None:
         if client is not None and remote_logs_handler is not None:
             remote_logs_handler.init_api_client(client)
 
+        if hasattr(args, "func"):
+            if args.subcommands == "upload-file":
```

Comment:
"else" what? It can't happen right? So instead
```suggestion
            assert args.subcommands == "upload-file"
```

Reply:
Are we sure there will be no other subcommands in the future?

Reply:
Once we add them, the assert will trigger and we will remove it :)